src/entitysdk/models/electrical_cell_recording.py

```diff
@@ -0,0 +1,86 @@
+"""Electrical cell recording models."""
+
+from typing import Annotated
+
+from pydantic import Field
+
+from entitysdk.models.brain_region import BrainRegion
+from entitysdk.models.entity import Entity
+from entitysdk.models.license import License
+from entitysdk.models.subject import Subject
+
+
+class ElectricalRecordingStimulus(Entity):
+    """Electrical cell recording stimulus model."""
+
+    dt: float | None = None
+    injection_type: str
+    shape: str
+    start_time: float | None = None
+    end_time: float | None = None
+
+
+class ElectricalCellRecording(Entity):
+    """Electrical cell recording model."""
+
+    ljp: Annotated[
+        float,
+        Field(
+            title="Liquid Junction Potential",
+            description="Correction applied to the voltage trace, in mV",
+            examples=[0.1],
+        ),
+    ] = 0.0
+    recording_location: Annotated[
+        list[str],
+        Field(
+            title="Recording Location",
+            description=(
+                "Location on the cell where recording was performed, in hoc-compatible format."
+            ),
+        ),
+    ]
+    recording_type: Annotated[
+        str,
+        Field(
+            title="Recording Type",
+            description="Recording type.",
+        ),
+    ]
+    recording_origin: Annotated[
+        str,
+        Field(
+            title="Recording Origin",
+            description="Recording origin.",
```

Comment:
Also here, listing the possible values would make clearer what we mean by origin:
"in_vivo", "in_vitro", "in_silico", "unknown"

Reply:
Could you also use the ElectricalRecordingOrigin here please?

Reply:
oops, missed that